spec/acceptance/radius_server_spec.rb

```diff
@@ -1,7 +1,9 @@
 require 'spec_helper_acceptance'
+require 'pry'
 
 describe 'radius_server' do
   before(:all) do
+    skip "this device #{device_model} does not support radius server" if %w(3750 6503).include?(device_model)
```

Comment:
brilliant that we can now call out exclusions in tests! :) 👍

Reply:
Can we use these opportunities to link to canonical docs having this information, like a feature matrix from cisco, or similar?